src/components/Article/Editor.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dante from 'Dante2';
+
+function Editor(props) {
```

Comment:
all functions should be written in es6 syntax

Reply:
And just curious, what purpose does this file serve in regards to your task?

Reply:
On it.

Reply:
I had to rebase onto the create article branch so that I could access the articles. This came from that branch.